imap_processing/cli.py

```diff
@@ -374,7 +380,19 @@ class Codice(ProcessInstrument):
     """Process CoDICE."""
 
     def do_processing(self, dependencies):
-        """Perform CoDICE specific processing."""
+        """
+        Perform CoDICE specific processing.
+
+        Parameters
+        ----------
+        dependencies : array_like
```

Comment:
```suggestion
        dependencies : list
```

I think we should just keep these is "list" to be consistent with how the rest of this code treats this `dependencies` data?